api/provisioner/provisioner.go

```diff
@@ -246,37 +246,12 @@ func (st *State) ReleaseContainerAddresses(containerTag names.MachineTag) (err e
 // PrepareContainerInterfaceInfo allocates an address and returns information to
 // configure networking for a container. It accepts container tags as arguments.
 func (st *State) PrepareContainerInterfaceInfo(containerTag names.MachineTag) (corenetwork.InterfaceInfos, error) {
-	return st.prepareOrGetContainerInterfaceInfo(containerTag, true)
-}
-
-// GetContainerInterfaceInfo returns information to configure networking
-// for a container. It accepts container tags as arguments.
-func (st *State) GetContainerInterfaceInfo(containerTag names.MachineTag) (corenetwork.InterfaceInfos, error) {
```

Comment:
This was never actually called by `MaintainInstance` and is unused, so the chicanery around `prepareOrGet...` booleans is moot.